### PR TITLE
Add support for presenting on window scenes

### PR DESCRIPTION
**Pull request by AvdLee**
iOS 13 introduced window scenes that weren't correctly handled by our framework yet. This PR makes it at least possible to present notifications when there's only 1 window.

Also updated the minimum iOS from 9 to 13 to keep things simple in maintaining the code.

### CODE REVIEW

File: Sources/UINotification.swift [hunk 1]
```diff
@@ -57,11 +57,15 @@ public final class UINotification: Equatable {
         case custom(height: CGFloat)
 
         internal var value: CGFloat {
+            guard let statusBarManager = UIApplication.shared.windows.first(where: \.isKeyWindow)?.windowScene?.statusBarManager else {
```
Comment (wetransferplatform) [inline Sources/UINotification.swift:60]: - :no_entry_sign: 'statusBarManager' is only available in iOS 13.0 or newer
- :no_entry_sign: 'windowScene' is only available in iOS 13.0 or newer

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Failed to get the coverage - Error: The data couldn’t be read because it isn’t in the correct format.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **Package.swift#L8** - Collection literals should not have trailing commas. (`trailing_comma`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **Sources/UINotificationCenter.swift#L37** - 'UIWindowScene' is only available in iOS 13.0 or newer
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **Sources/UINotificationCenter.swift#L37** - 'connectedScenes' is only available in iOS 13.0 or newer
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **Sources/UINotificationCenter.swift#L38** - 'init(windowScene:)' is only available in iOS 13.0 or newer
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationCenter.swift#L37** - Prefer using `.first(where:)` over `.filter { }.first` in collections. (`first_where`)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/32a8785c-443f-4a8a-beed-f1ee5fe5c555" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against c5552b211eb4041cef56dfe1c3266b5b6bebe7ed
</p>

File: Sources/UINotificationCenter.swift [hunk 2]
```diff
@@ -32,22 +32,29 @@ public final class UINotificationCenter {
     
     /// The window which will be placed on top of the application window.
     /// This window is used to present the notifications on and will be hidden when notifications are dismissed.
-    internal let window: UIWindow
+    internal lazy var window: UIWindow = {
+        let window: UIWindow
+        if let windowScene = UIApplication.shared.connectedScenes.filter({ $0.activationState == .foregroundActive }).first as? UIWindowScene {
```
Comment (wetransferplatform) [inline Sources/UINotificationCenter.swift:37]: - :no_entry_sign: 'UIWindowScene' is only available in iOS 13.0 or newer
- :no_entry_sign: 'connectedScenes' is only available in iOS 13.0 or newer
- :warning: Prefer using `.first(where:)` over `.filter { }.first` in collections. (`first_where`)

File: Sources/UINotificationCenter.swift [hunk 3]
```diff
@@ -32,22 +32,29 @@ public final class UINotificationCenter {
...
-    /// Creates a new notification center. Sets up the window for usage.
-    internal init() {
-        window = UINotificationPresentationWindow()
-        window.clipsToBounds = true
-        window.isUserInteractionEnabled = true
-        window.backgroundColor = UIColor.clear
-        window.windowLevel = UIWindow.Level.normal - 1
-    }
+    /// Creates a new notification center.
+    internal init() { }
```
Comment (kairadiagne) [inline Sources/UINotificationCenter.swift:57]: Nit: Is this init still needed. Don't we already have this one by default?

**Review by kairadiagne — CHANGES_REQUESTED** [review]
Looks good, I got only one question.

File: Sources/UINotificationCenter.swift (hunk 3 above)
Comment (AvdLee) [reply Sources/UINotificationCenter.swift:57]: You're right, good point!

**Review by kairadiagne — APPROVED** [review]

File: Package.swift [hunk 4]
```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "UINotifications",
     platforms: [
-        .iOS(.v9),
+        .iOS(.v13),
```
Comment (wetransferplatform) [inline Package.swift:8]: - :no_entry_sign: Collection literals should not have trailing commas. (`trailing_comma`)

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 1.6.0](https://github.com/WeTransfer/UINotifications/releases/tag/1.6.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**